_data/companies/opslock.yml

```
@@ -1,7 +1,7 @@
 ---
 name: Opslock
 wfh: Strongly Encouraged
-travel: "?"
-visitors: "?"
-events: "?"
-last_update: 2020-03-13
+travel: "Discouraged"
+visitors: "Discouraged"
+events: "Discouraged"
```

```suggestion
travel: Discouraged
visitors: Discouraged
events: Discouraged
```

The quotes are only needed because ? is a special character in YAML.